website/static/playground.js

```diff
@@ -496,18 +496,27 @@ function getExample(parser) {
       ].join("\n");
     case "markdown":
       return [
+        "Header",
+        "======",
+        "",
         "_Look,_ code blocks are formatted *too!*",
         "",
         "``` js",
         "function identity(x) { return x }",
         "```",
         "",
+        "Pilot|Airport|Hours",
+        "--|:--:|--:",
+        "John Doe|SKG|1338",
+        "Jane Roe|JFK|314",
+        "",
+        "- - - - - - - - - - - - - - -",
+        "",
         "+ List",
         " + with a [link] (/to/somewhere)",
-        "+ and [another one]",
-        "",
         "",
-        "  [another one]:  http://example.com 'Example title'"
```

Comment:
This one is the example for `--single-quote`, we shouldn't remove it.

Reply:
Right!